tests/e2e/tests/pilot/testdata/networking/v1alpha3/wildcard-https-google-no-vs.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: wildcardgooglenovs
+spec:
+  hosts:
+  - "*.google.co.uk"
```

Comment:
replace with yahoo.. protocol HTTPS. We are already testing TLS protocol in other existing service entries

Reply:
yes I am using https protocol here.  I had *.google.co.uk or *.google.co.in.  Any advantage of using yahoo over google?  it seems same to me and google has more availability imho.